test/logstash/snippets/vcap-spec.rb

```diff
@@ -69,7 +69,7 @@
     sample("@type" => "relp", "host" => "1.2.3.4", "@message" => '<78>Apr 24 04:03:06 localhost crontab[32185]: (root) LIST (root)') do
       insist { subject["tags"] } == [ 'syslog_standard' ]
       insist { subject["@type"] } == 'relp'
-      insist { subject["@timestamp"] } == Time.iso8601("2014-04-24T04:03:06.000Z")
+      insist { subject["@timestamp"] } == Time.iso8601("2015-04-24T04:03:06.000Z")
```

Comment:
@devcamcar - thanks!

however, would this fix not also fail next year?

Could you update this PR so it checks for `Time.iso8601("#{current_year}-04-24T04:03:06.000Z")` please